src/validation/tests/duplicates_validation_test.rs

```diff
@@ -1,14 +1,17 @@
 use crate::{
     assert_validation_snapshot,
-    ast::{self, CompilationUnit, SourceRangeFactory},
     index::{visitor, Index},
-    lexer::{self, IdProvider},
+    lexer::{self},
```

Comment:
Again, this is not really an issue, but this could just be `lexer`

Reply:
Good catch, weird that `cargo fmt` doesn't do this automatically tho